internal/compiler/passes/visible.rs

```diff
@@ -67,10 +67,14 @@ pub fn handle_visible(
                         clip_elem.borrow_mut().bindings.remove("width");
                         clip_elem.borrow_mut().bindings.remove("height");
                     }
-                } else if has_visible_binding(&child) {
-                    let new_child = create_visibility_element(&child, &native_clip);
-                    new_child.borrow_mut().children.push(child);
-                    child = new_child;
+                } else if child.borrow().repeated_as_embedding().is_some() {
+                    todo!()
```

Comment:
we could potentially special case the visible property for Embed, but it would be set on the parent, not the child